src/raster/rasterband.rs

```diff
@@ -320,3 +333,13 @@ impl<T: GdalType> Buffer<T> {
 }
 
 pub type ByteBuffer = Buffer<u8>;
+
+pub fn get_color_interpretation_name(color_interp: GDALColorInterp::Type) -> String {
```

Comment:
I'm just a drive by reviewer, but my two cents:

Typically we've been creating wrapper structs for the underlying gdal-sys types, rather than adding free methods. Would it be more in line with the rest of the crate to have an API more like:

```
struct ColorInterpretation(GDALColorInterp::Type)

impl ColorInterpretation {
    fn name(&self) -> &str { ... }
    fn from_name(&str) -> Self  { ... }
}

Reply:
Sounds reasonable. I guess it should be an enum with interpretations then